src/server.ts

```diff
@@ -18,15 +20,21 @@ export function start(): Promise<ServerInfo> {
                 const serverLocation = path.resolve(__dirname, '..', '..', 'server');
                 const felix = path.join(serverLocation, 'bin', 'felix.jar');
                 const java = path.join(home, 'bin', 'java');
-                const process = cp.spawn(java, ['-jar', felix], { cwd: serverLocation });
-                waitOn({
-                    resources: ['tcp:localhost:27511']
-                }, () => {
-                    resolve({
-                        port: 27511,
-                        host: 'localhost',
-                        process
+                portfinder.getPortPromise()
```

Comment:
set portfinder.basePort = 27511 before calling getPortPromise(), so it wiuld start search form our current hardcoded port.